web/api/queries/query.c

```diff
@@ -1419,17 +1412,6 @@ static RRDR *rrd2rrdr_variablestep(
     if(before_wanted < first_entry_t)
         error("INTERNAL CHECK: before_wanted %u is too small, minimum %u", (uint32_t)before_wanted, (uint32_t)first_entry_t);
 
-    if(before_wanted > last_entry_t)
-        error("INTERNAL CHECK: before_wanted %u is too big, maximum %u", (uint32_t)before_wanted, (uint32_t)last_entry_t);
-
-/*
-    if(before_slot >= (size_t)st->entries)
-        error("INTERNAL CHECK: before_slot is invalid %zu, expected 0 to %ld", before_slot, st->entries - 1);
-
-    if(after_slot >= (size_t)st->entries)
-        error("INTERNAL CHECK: after_slot is invalid %zu, expected 0 to %ld", after_slot, st->entries - 1);
-*/
-
```

Comment:
Hi Stelio. Since this part was removed, should also the part in https://github.com/netdata/netdata/blob/3dbe22e89e8672758f6bfcbe2e5f8c357010b6b6/web/api/queries/query.c#L1033-L1042 be removed too?

Reply:
yes it is good to remove -- I will remove it